MVP/run

```diff
@@ -1,65 +1,71 @@
 #!/bin/bash
 
+# Function to convert Windows path to Unix path
+convert_path() {
+  echo "$1" | sed 's/\\/\//g'
+}
+
 case "$1" in
-  install)
-    echo "Installing dependencies..."
-    npm install
-    exit $?
-    ;;
-  
-  test)
-    echo "Installing dependencies..."
-    npm install
-    
-    echo "Building the project..."
-    npm run build
-    if [ $? -ne 0 ]; then
-      echo "Build failed."
-      exit 1
-    fi
+install)
+  npm install >/dev/null 2>&1
+  npm run build >/dev/null 2>&1 # suppressing output
+  node -e "require('./dist/Output/CLI').printDependencyCount()"
```

Comment:
if you call 

printDependencyCount()

in this file, you can use

node ./dist/Output/CLI.js